### PR TITLE
Fixing margin on About Hero section. 

### PR DESCRIPTION
**Pull request by codiejay**
**### Before adjusting margins:** 
image
 
### **After Adjusting margins:**  
image

## Linked issues

closes #330, closes #330, closes https://github.com/EveripediaNetwork/issues/issues/330

### CODE REVIEW

**Comment by Emmanueldmlr** [comment]
The blog section looks shrank.

image

**Comment by Emmanueldmlr** [comment]
They are all supposed to be horizontally displayed.
image

**Comment by Emmanueldmlr** [comment]
The hero margin is a bit off on a larger screen 

image

**Comment by Emmanueldmlr** [comment]
@codiejay, the margin is off on small screen 
image

and also on medium screen
image

**Comment by codiejay** [comment]
> @codiejay, the margin is off on small screen image
> 
> and also on medium screen image

oh that's weird @Emmanueldmlr , cause I have a different view on my local. 
Let me fix that

**Comment by Emmanueldmlr** [comment]
@codiejay, can you add a margin-top to the hero and space the hero elements out on a smaller and medium screen...Am I right @Simplypheyie96?

image

**Comment by Simplypheyie96** [comment]
> @codiejay, can you add a margin-top to the hero and space the hero elements out on a smaller and medium screen...Am I right @Simplypheyie96?
> 
> image

Yes you are.

**Comment by kesar** [comment]
@codiejay make sure in the PR description you attach images of before / after and different screens. That way its easier for reviewers 👍🏻 and that will force you to make sure it works fine before pushing it for reviewing.

Take as inspiration:
https://github.com/EveripediaNetwork/ep-ui/pull/257
https://github.com/EveripediaNetwork/ep-ui/pull/280
https://github.com/EveripediaNetwork/ep-ui/pull/279
https://github.com/EveripediaNetwork/ep-ui/pull/275

**Comment by Simplypheyie96** [comment]
Please do tag me for reviews so i can point out mistakes in ui @codiejay

**Comment by Emmanueldmlr** [comment]
@codiejay , the 3rd blog card seems broken...you can inspect to know what's wrong, maybe the length of the text or the height of the card.
image

**Comment by Emmanueldmlr** [comment]
@codiejay, this is what it used to be like.
image

**Comment by Emmanueldmlr** [comment]
That will be all from my side. @Simplypheyie96, do you have any other issues you think he should correct?.

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
image
This image is for the dark mode.

This is for the light mode. you will see it in the zip file I sent
image

**Comment by Simplypheyie96** [comment]
As seen in section should be this way 
image
@codiejay

**Comment by Simplypheyie96** [comment]
The font sizes of sub-texts are too small and fonts in general are not in alignment with the homepage of everipedial

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
Is there something we can do to make this not stretched? When i changed the zoom, i discovered that it shrinks as the size gets smaller. can we do something about that? Other cards don't behave that way.
image

Also, is there a way to not make the words sit atop each other? It shouldn't be more than 2 lines.
image

File: src/components/About/AboutOurTeam.tsx [hunk 1]
```diff
@@ -37,77 +38,82 @@ const IconButtonSocial = ({
...
+              display="block !important"
+              mx="auto"
+              key={teamMember.name}
+              alignItems="center"
+              justifyContent="center"
+            >
+              <Box>
+                <Image
+                  w="150px"
+                  h="150px"
```
Comment (anubra266) [inline src/components/About/AboutOurTeam.tsx:67]: You can do
```suggestion
                  boxSize="150px"
```

**Review by Simplypheyie96 — APPROVED** [review]
It looks good now. Good to go!

**Pull request merged**